allennlp/data/tokenizers/word_splitter.py

```diff
@@ -15,16 +16,15 @@ class WordSplitter(Registrable):
     """
     default_implementation = 'spacy'
 
-    def split_words(self, sentence: str) -> Tuple[List[str], List[Tuple[int, int]]]:
+    def split_words(self, sentence: str) -> List[Token]:
         """
         Splits ``sentence`` into tokens, returning the resulting tokens and the character offsets
         for each token in the original string.  Not all ``WordSplitters`` implement character
         offsets, so the second item in the returned tuple could be ``None``.
```

Comment:
fix comment, still refers to "returned tuple"

Reply:
Thanks, done.